src/test/java/com/rewedigital/composer/CompositionTest.java

```diff
@@ -0,0 +1,70 @@
+package com.rewedigital.composer;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.CompletionStage;
+import java.util.concurrent.ExecutionException;
+
+import org.junit.Rule;
+import org.junit.Test;
+
+import com.spotify.apollo.Response;
+import com.spotify.apollo.test.ServiceHelper;
+import com.squareup.okhttp.mockwebserver.MockResponse;
+import com.squareup.okhttp.mockwebserver.MockWebServer;
+
+import okio.ByteString;
+
+public class CompositionTest {
+
+    @Rule
+    public final MockWebServer server = new MockWebServer();
+
+    @Rule
+    public ServiceHelper serviceHelper =
+        ServiceHelper.create(ComposerApplication.Initializer::init, ComposerApplication.COMPOSER)
+            .conf("composer.routing.local-routes", routesConfig(mockServerUrl()))
```

Comment:
After you have introduced the constant `ComposerApplication.COMPOSER`, shouldn't you use it here? But on the other hand, `ComposerApplication.COMPOSER + "routing.local-routes"` is so much worse regarding readability, that I don't like the constant at all ;-)